### DISCUSSION TITLE
GitHub site can't access

### DISCUSSION THREAD

**Discussion by RishikeshSharmak** [community/community#38769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

https://user-images.githubusercontent.com/89477156/201082749-4a43227e-3bb2-4e91-81aa-6390bc15f47f.mp4

Hey ,I'm facing problem opening the GitHub site in my laptop,
In normal tab it not opening it showing error
Err_address_invalid but in brave browser with private tab with tor, it opening the website 
And  when I opened in my another laptop with same isp connection(no private window)it `working...
![16680807752348670834873471111892](https://user-images.githubusercontent.com/89477156/201082642-44d36375-8016-4fe6-84ef-cfe449cf53d1.jpg)
`
Please help me
My account is not banned
Is my laptop serial number or laptop banned?

**Comment by airtower-luna** [comment]
That's clearly a DNS problem, your browser can't find the IP address for `github.com`. Tor does its own DNS resolution, so it will get around local DNS problems as long as it can still get a connection.

You'll have to investigate your local network to figure out the problem (DNS settings on your computer, router, etc.), or possibly contact your provider if the problem is outside those.

As a quick fix, you could try to configure an alternative DNS server (e.g. [Quad9](https://quad9.net/)).

**Reply by RishikeshSharmak** [reply]
Hey thank you for responding 
I done above mentioned steps, i changed dns to quad9 but still same error.. and i don't think it internet service provider problem because i connected to same network same wifi to my another laptop, i can open GitHub..

It can be firewall?blocking server access or something?
I done disabling and restoring windows firewall but didn't fixed , same problem 😔

https://user-images.githubusercontent.com/89477156/201122393-7283363e-006e-4563-9328-eabb20cbe771.mp4

 fixed yet..

**Reply by airtower-luna** [reply]
If other devices on the same network work then yes, it's probably not a provider or router problem.

That means you'll have to take a closer look at your computer: Which DNS server does it actually use? Are there any proxy settings? Do you use a VPN (which could be misconfigured)? And so on. I can't look into your computer, so I can't say what it might be.

**Reply by RishikeshSharmak** [reply]
Hey once again thank you for responding
I fixed, My hosts file was having some problems,corrupted may be, 
So i just restored my hosts file content to default,restored....

**Reply by ashanviii** [reply]
> Hey once again thank you for responding I fixed, My hosts file was having some problems,corrupted may be, So i just restored my hosts file content to default,restored....

Hey how did you fix it can you explain it to me?

**Comment by 03timothyash** [comment]
Ok

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 03timothyash** [comment]
Claro que. Si podría ser es necesaria una marca para que registre. Y terminar el juego de este proseso

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
